selfservice/flow/request_test.go

```diff
@@ -101,74 +101,3 @@ func TestMethodEnabledAndAllowed(t *testing.T) {
 		assert.Contains(t, string(body), "The requested resource could not be found")
 	})
 }
-
-func TestMethodCodeEnabledAndAllowed(t *testing.T) {
```

Comment:
Has this test moved somewhere else? I think that would be important to ensure we don't have regressions.

Reply:
This test only tested the special handling of `passwordless_enabled` in the method, which I removed. The `MethodEnabledAndAllowed` method is tested in `TestMethodEnabledAndAllowed` in this file. 

The code configuration is covered in the registry tests: https://github.com/ory/kratos/blob/8821e6b0c51f7fbec05e05712ec4d7b53647aba4/driver/registry_default_test.go#L618